components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/micro/integrator/inbound/endpoint/protocol/grpc/util/Event.java

```diff
@@ -0,0 +1,818 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Shall we please add the licence header for this generated file

Reply:
@Nirothipan fixed